### DISCUSSION TITLE
Should font (or any CSS style) applied to a parent be visible in its children's style?

### DISCUSSION THREAD

**Discussion by darshitac11** [Atri-Labs/atrilabs-engine#160] [Discuss: New Feature Ideas]
I'm putting this up for discussion especially for font.  

**Background:**
I had applied a font to the parent Flex container to avoid having to apply it to each child text box. However, for one text box, I wanted the font style to be different from the one applied to the parent. 

**Problem:**
However, in child's style panel, there were no options available for style since font is inherited form parent. So, in order to change style, I had to reapply the same font (which is what I wanted to avoid in the first place) and then change style?

![Discussion#160](https://user-images.githubusercontent.com/102641692/189522763-36ed0375-4a36-443a-bdb9-a91d07b179f1.gif)

**Comment by cruxcode** [comment]
Yes, we should implement this as this will make the developer's life easier. We can open an issue for this, but before that, we will need a design for how will we show the inherited properties.

**Comment by darshitac11** [comment]
- Add a section in the top of style panel. 

- This section launches a popup which opens over the canvas.

- This popup has two sections: 
1. The overrides
2. The inherited properties

Refer browser's styles section for design.
